bin/aws

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Does this script download everytime  or only the first time?

Reply:
Only the first time. The aws installer overwrites this file with a symlink automatically.